### PR TITLE
readme

### PR DESCRIPTION
**Pull request by alsgregory**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
 # flooddrake
 Firedrake DG shallow water model with wetting and drying for flood simulation
+
+## Linux terminal installation
+As a prerequisite, it is required to have Firedrake installed alongside it's dependencies. Instructions on how to this can be found at: http://firedrakeproject.org/download.html.
+
+To install, type the following commands into the terminal whilst in an empty repository:
+
+1. `git init`
```
Comment (miklos1) [inline README.md]: What's the point of this line?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
 # flooddrake
 Firedrake DG shallow water model with wetting and drying for flood simulation
+
+## Linux terminal installation
+As a prerequisite, it is required to have Firedrake installed alongside it's dependencies. Instructions on how to this can be found at: http://firedrakeproject.org/download.html.
+
+To install, type the following commands into the terminal whilst in an empty repository:
```
Comment (miklos1) [inline README.md]: What do you mean "whilst in an empty repository"?

Comment (miklos1) [inline README.md]: You should emphasise that the Firedrake virtualenv must be activated.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,15 @@
...
+
+1. `git init`
+2. `git clone https://github.com/firedrakeproject/flooddrake flooddrake`
+3. `cd ./flooddrake`
+4. `pip install .`
```
Comment (miklos1) [inline README.md]: You will make upgrades easier for your users with an editable install, that is `pip install -e .`

Comment (miklos1) [inline README.md]: You can collapse 3. and 4. into `pip install -e ./flooddrake`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,15 @@
 # flooddrake
 Firedrake DG shallow water model with wetting and drying for flood simulation
+
+## Linux terminal installation
+As a prerequisite, it is required to have Firedrake installed alongside it's dependencies. Instructions on how to this can be found at: http://firedrakeproject.org/download.html.
+
+To install, type the following commands into the terminal whilst in an empty repository:
+
+1. `git init`
+2. `git clone https://github.com/firedrakeproject/flooddrake flooddrake`
```
Comment (miklos1) [inline README.md]: You don't necessarily need the last argument (flooddrake) as that will be the default directory name anyway, as it matches the last part of the URL.

**Comment by alsgregory** [comment]
Sorry shouldn't have pushed. Updates made in #19

**Pull request merged**